### DISCUSSION TITLE
Receive a student discount on a different account.

### DISCUSSION THREAD

**Discussion by mshogo6** [community/community#56807] [GitHub Education]
### Hi everyone,

How can I cancel the student discount once received and receive it again on a different account?

When I look at the account I am currently receiving the discount on, it appears that I have less than one year left on the discount.
When I try to submit a request for a new account, it says "_NEW ACCOUNT_ has already been used to obtain an academic discount on GitHub account(s) _CURRENT ACCOUNT_" and the request does not go through.
I think I need to cancel the remaining privileges for less than a year before you can apply.

How can I do this?

**Comment by ghost** [comment]
same

**Comment by mshogo6** [comment]
It does not appear that you can transfer your GitHub Student Developer Pack membership to another personal account.
It must be merged into an account that has an academic discount applied to it.
